kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/scheduler/KafkaScheduler.java

```diff
@@ -180,40 +178,37 @@ private void startApiServer() {
             }
         });
     }
+    // dcos-commons 0.8.1 upgrade: just to be consistent with the existing java tests (see KafkaSchedulerTest.java)
```

Comment:
This is a one line function. Comment is a reminder for that and will remove createDeployPlanManager totally.